scripts/zones/AlTaieu/mobs/Jailer_of_Prudence.lua

```diff
@@ -33,43 +42,91 @@ entity.onMobInitialize = function(mob)
 end
 
 entity.onMobSpawn = function(mob)
-    xi.mix.jobSpecial.config(mob, {
-        specials =
-        {
-            {
-                id = xi.jsa.PERFECT_DODGE,
-                cooldown = 120, -- "Both can use Perfect Dodge multiple times, and will do so almost incessantly." (guessing a 2 minute cooldown)
-                hpp = 95,
-                endCode = function(mobArg)
-                    mobArg:addStatusEffectEx(xi.effect.FLEE, 0, 10000, 0, 30) -- "Jailer of Prudence will however gain Flee speed during Perfect Dodge."
-                end,
-            },
-        },
-    })
-
     mob:setAnimationSub(0) -- Mouth closed
-    mob:addStatusEffectEx(xi.effect.FLEE, 0, 10000, 0, 60)
     mob:setMod(xi.mod.TRIPLE_ATTACK, 20)
-    mob:setMod(xi.mod.REGEN, 10)
-    mob:addMod(xi.mod.BIND_MEVA, 30)
-    mob:addMod(xi.mod.SLOW_MEVA, 10)
-    mob:addMod(xi.mod.BLIND_MEVA, 10)
-    mob:addMod(xi.mod.SLEEP_MEVA, 30)
-    mob:addMod(xi.mod.PETRIFY_MEVA, 10)
-    mob:addMod(xi.mod.GRAVITY_MEVA, 10)
-    mob:addMod(xi.mod.LULLABY_MEVA, 30)
+    mob:addMod(xi.mod.DEFP, 33)
+    mob:addMod(xi.mod.MOVE_SPEED_STACKABLE, 20)
+    mob:addMobMod(xi.mobMod.WEAPON_BONUS, 72) -- 180 total weapon damage
+end
+
+local teleportToTarget = function(mob, target, currentTargetId)
+    mob:setMobMod(xi.mobMod.NO_MOVE, 1)
+    mob:timer(500, function(mobArg)
+        if not mobArg then
+            return
+        end
+
+        local targetX, targetY, targetZ = target:getXPos(), target:getYPos(), target:getZPos()
+        local offsetX = math.random(-2, 2)
+        local offsetZ = math.random(-2, 2)
+        mobArg:setStatus(xi.status.INVISIBLE)
+        mobArg:setPos(targetX + offsetX, targetY, targetZ + offsetZ, target:getRotPos())
+
+        mobArg:timer(500, function(mobArg2)
+            if not mobArg2 then
+                return
+            end
+
+            mobArg2:setStatus(xi.status.UPDATE)
+            mobArg2:setMobMod(xi.mobMod.NO_MOVE, 0)
+        end)
+    end)
 end
 
-entity.onMobDisengage = function(mob)
+entity.onMobFight = function(mob, target)
+    -- Manual Perfect Dodge control to sync up usage between both Prudences
+    local now = GetSystemTime()
+    local prudenceId = ID.mob.JAILER_OF_PRUDENCE
+    local otherPrudence = mob:getID() == prudenceId and GetMobByID(prudenceId + 1) or GetMobByID(prudenceId)
+
+    local hasUsed = mob:getLocalVar('perfectDodgeUsed') == 1
+    local cooldownTime = mob:getLocalVar('perfectDodgeCooldown')
+
+    if
+        (not hasUsed and mob:getHPP() <= 95) or
+        (hasUsed and now >= cooldownTime)
+    then
```

Comment:
Ok, in general, please use verbose variable names.

This code has to be able to be used by the whole comunity.
We dont need "now" or "hasUsed". We need "currentTime" and "hasUsedPerfectDodge" and such.
Saving a few clicks just isn't worth it if an snippet cant be understood at a glance.